examples/child_window.rs

```diff
@@ -0,0 +1,83 @@
+#[cfg(all(target_os = "linux", feature = "x11"))]
+use std::collections::HashMap;
+
+#[cfg(all(target_os = "linux", feature = "x11"))]
+use winit::{
+    dpi::{LogicalPosition, LogicalSize, Position},
+    event::{ElementState, Event, KeyboardInput, WindowEvent},
+    event_loop::{ControlFlow, EventLoop, EventLoopWindowTarget},
+    platform::unix::{WindowBuilderExtUnix, WindowExtUnix},
+    window::{Window, WindowBuilder},
+};
+
+#[cfg(all(target_os = "linux", feature = "x11"))]
+fn spawn_child_window(
+    parent: usize,
+    event_loop: &EventLoopWindowTarget<()>,
+    windows: &mut HashMap<usize, Window>,
+) {
+    let child_window = WindowBuilder::new()
+        .with_x11_parent(parent)
+        .with_title("child window")
+        .with_inner_size(LogicalSize::new(200.0f32, 200.0f32))
+        .with_position(Position::Logical(LogicalPosition::new(0.0, 0.0)))
```

Comment:
Is this position relative to the parent's inner coordinate system or the screen or something else?

Reply:
The position is relative to parent's inner one.

Reply:
Updating docs that is very nice idea. I will update it please wait.

Reply:
How about this https://github.com/rust-windowing/winit/pull/2246/commits/daf680c87aec2d4f666c4fa119a206a9cc213a31 ?